Backend/routes/user.js

```diff
@@ -5,7 +5,7 @@ import { uploadTwitterProImg } from "../middlewares/multer.js";
 
 const userRouter = (router)=>{
     router.get('/whotofollow',userController.getWhoToFollow)
-    router.patch('/hanldefollow',userController.handleFollowUnfollow)
+    router.post('/hanldefollow',userController.handleFollowUnfollow)
```

Comment:
for endpoints having multiple words use - as separator for readability

Reply:
used hyphens as separators for multi-word endpoints.you can see the updates in the latest commit.